website/docs/reference/global-configs/version-compatibility.md

```diff
@@ -12,4 +12,9 @@ You can use the `VERSION_CHECK` config to disable this check and suppress the er
 dbt --no-version-check run
 Running with dbt=1.0.0
 Found 13 models, 2 tests, 1 archives, 0 analyses, 204 macros, 2 operations....
-```
\ No newline at end of file
+```
+
+:::info Keep with latest version
+<Snippet path="_config-dbt-version-check" />
+
+:::
```

Comment:
i waffled between a callout and subsection -- i think either can work imo

Reply:
I think it works here and now, but maybe it should be temporary and sometime after GA change it to a section.

Reply:
exactly! i was thinking the same thing